### PR TITLE
Refs #60596 - Adds helper + CSS for displaying images in list views

### PR DESCRIPTION
**Pull request by aaronransley**
* Issue: https://issues.afinedevelopment.com/issues/60596

### CODE REVIEW

File: app/helpers/fae/view_helper.rb [hunk 1]
```diff
@@ -25,6 +25,17 @@ def fae_content_form(f, attribute, label: nil, hint: nil, helper_text: nil, mark
...
 
+    def fae_index_image(image, path = nil)
+      return if image.blank? || image.asset.blank? || image.asset.thumb.blank?
+      output = nil
+      if path
+        output = link_to path do image_tag(image.asset.thumb.url) end
+      else
+        output = image_tag(image.asset.thumb.url)
+      end
+      "<div class='image-mat'>#{output}</div>".html_safe
```
Comment (tshedor) [inline app/helpers/fae/view_helper.rb]: This should utilize the `content_tag` Rails helper.

File: app/helpers/fae/view_helper.rb [hunk 2]
```diff
@@ -25,6 +25,17 @@ def fae_content_form(f, attribute, label: nil, hint: nil, helper_text: nil, mark
       render 'fae/application/content_form', f: f, attribute: attribute, label: label, hint: hint, helper_text: helper_text, markdown: markdown, markdown_supported: markdown_supported, input_options: input_options
     end
 
+    def fae_index_image(image, path = nil)
+      return if image.blank? || image.asset.blank? || image.asset.thumb.blank?
+      output = nil
```
Comment (tshedor) [inline app/helpers/fae/view_helper.rb]: You'll always have `output`; this line is unnecessary.

File: app/helpers/fae/view_helper.rb [hunk 3]
```diff
@@ -25,6 +25,17 @@ def fae_content_form(f, attribute, label: nil, hint: nil, helper_text: nil, mark
...
+      if path
```
Comment (tshedor) [inline app/helpers/fae/view_helper.rb]: I think what you're looking for is [`link_to_if`](http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to_if).

File: app/assets/stylesheets/fae/modules/tables/_image.scss [hunk 4]
```diff
@@ -0,0 +1,19 @@
+th {
+  &.-image {
+    width: 120px;
+  }
+}
+
+td {
+  .image-mat {
```
Comment (tshedor) [inline app/assets/stylesheets/fae/modules/tables/_image.scss]: This property doesn't need to be nested. Additionally, in `_actions.scss`, there's a rule for `&.-thumb` on nested table columns that have an image. Move that rule into this file please, for consistency's sake.

File: app/helpers/fae/view_helper.rb (hunk 3 above)
Comment (jamesmk) [inline app/helpers/fae/view_helper.rb]: Usually not a fan of that helper, but think it's a good fit here.

**Comment by jamesmk** [comment]
@aaronransley Can you document the the helper in `view_helpers.md` and add a couple tests for it in `fae_view_helper_spec.rb`?

**Comment by aaronransley** [comment]
Tests + documentation added. Integrated feedback from @tshedor to great ends; we're left with a very lean helper method 👍

**Pull request merged**